<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1180</width>
    <height>728</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <widget class="QPushButton" name="exitBtn">
    <property name="geometry">
     <rect>
      <x>850</x>
      <y>650</y>
      <width>184</width>
      <height>73</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(255, 255, 255, 0);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="../resources/images/images.qrc">
      <normaloff>:/images/screens/button_exit.png</normaloff>:/images/screens/button_exit.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>184</width>
      <height>73</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="newGameBtn">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>41</x>
      <y>613</y>
      <width>91</width>
      <height>90</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(255, 255, 255, 0);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="../resources/images/images.qrc">
      <normaloff>:/images/screens/button_ng.png</normaloff>:/images/screens/button_ng.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>91</width>
      <height>90</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="pauseBtn">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>153</x>
      <y>613</y>
      <width>91</width>
      <height>90</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(255, 255, 255, 0);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="../resources/images/images.qrc">
      <normaloff>:/images/screens/button_pause.png</normaloff>:/images/screens/button_pause.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>91</width>
      <height>90</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="resumeBtn">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>267</x>
      <y>613</y>
      <width>91</width>
      <height>90</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(255, 255, 255, 0);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="../resources/images/images.qrc">
      <normaloff>:/images/screens/button_resume.png</normaloff>:/images/screens/button_resume.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>91</width>
      <height>90</height>
     </size>
    </property>
   </widget>
   <widget class="QProgressBar" name="amPB">
    <property name="geometry">
     <rect>
      <x>376</x>
      <y>629</y>
      <width>118</width>
      <height>23</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>12</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="value">
     <number>24</number>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="textVisible">
     <bool>true</bool>
    </property>
    <property name="format">
     <string>%v</string>
    </property>
   </widget>
   <widget class="QProgressBar" name="hpPB">
    <property name="geometry">
     <rect>
      <x>376</x>
      <y>683</y>
      <width>118</width>
      <height>23</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="value">
     <number>24</number>
    </property>
    <property name="textVisible">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="heroBtn">
    <property name="geometry">
     <rect>
      <x>567</x>
      <y>611</y>
      <width>88</width>
      <height>109</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="../resources/images/images.qrc">
      <normaloff>:/images/heroes/0.png</normaloff>:/images/heroes/0.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>88</width>
      <height>109</height>
     </size>
    </property>
   </widget>
   <widget class="QProgressBar" name="levelPB">
    <property name="geometry">
     <rect>
      <x>53</x>
      <y>9</y>
      <width>290</width>
      <height>23</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="value">
     <number>24</number>
    </property>
    <property name="textVisible">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="statusLbl">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>6</y>
      <width>66</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="../resources/images/images.qrc">:/images/screens/score.png</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="levelLbl">
    <property name="geometry">
     <rect>
      <x>468</x>
      <y>7</y>
      <width>35</width>
      <height>38</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>16</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">background: transparent; color: white;</string>
    </property>
    <property name="text">
     <string>0</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="userLbl">
    <property name="geometry">
     <rect>
      <x>696</x>
      <y>0</y>
      <width>305</width>
      <height>44</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>24</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">background: transparent; color: white;</string>
    </property>
    <property name="text">
     <string>username</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="bestLbl">
    <property name="geometry">
     <rect>
      <x>930</x>
      <y>166</y>
      <width>50</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>12</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">background: transparent; color: black;</string>
    </property>
    <property name="text">
     <string>0</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="savedLbl">
    <property name="geometry">
     <rect>
      <x>891</x>
      <y>193</y>
      <width>50</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>12</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">background: transparent; color: black;</string>
    </property>
    <property name="text">
     <string>0</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="goldLbl">
    <property name="geometry">
     <rect>
      <x>816</x>
      <y>235</y>
      <width>140</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">background: transparent; color: yellow;</string>
    </property>
    <property name="text">
     <string>0</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="shopBtn">
    <property name="geometry">
     <rect>
      <x>1030</x>
      <y>209</y>
      <width>83</width>
      <height>63</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(255, 255, 255, 0);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="../resources/images/images.qrc">
      <normaloff>:/images/screens/button_shop.png</normaloff>:/images/screens/button_shop.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>83</width>
      <height>63</height>
     </size>
    </property>
   </widget>
   <widget class="QLabel" name="jumpCostLbl">
    <property name="geometry">
     <rect>
      <x>1044</x>
      <y>290</y>
      <width>50</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>12</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">background: transparent; color: black;</string>
    </property>
    <property name="text">
     <string>5C</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="homeCostLbl">
    <property name="geometry">
     <rect>
      <x>1044</x>
      <y>358</y>
      <width>50</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>12</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">background: transparent; color: black;</string>
    </property>
    <property name="text">
     <string>FREE!</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="aiCostLbl">
    <property name="geometry">
     <rect>
      <x>1044</x>
      <y>427</y>
      <width>50</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>12</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">background: transparent; color: black;</string>
    </property>
    <property name="text">
     <string>FREE!</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QTextBrowser" name="infoTB">
    <property name="geometry">
     <rect>
      <x>738</x>
      <y>519</y>
      <width>401</width>
      <height>97</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>12</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background: transparent; color: black; border: 0;</string>
    </property>
    <property name="lineWrapMode">
     <enum>QTextEdit::NoWrap</enum>
    </property>
    <property name="html">
     <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;meta charset=&quot;utf-8&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
hr { height: 1px; border-width: 0; }
li.unchecked::marker { content: &quot;\2610&quot;; }
li.checked::marker { content: &quot;\2612&quot;; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'微软雅黑'; font-size:12pt; font-weight:700; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'Segoe UI';&quot;&gt;Line 1&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'Segoe UI';&quot;&gt;Line 2&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'Segoe UI';&quot;&gt;Line 3&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'Segoe UI';&quot;&gt;Line 4&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="acceptRichText">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="jumpBtn">
    <property name="geometry">
     <rect>
      <x>771</x>
      <y>272</y>
      <width>48</width>
      <height>56</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background: transparent;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="../resources/images/images.qrc">
      <normaloff>:/images/screens/set.png</normaloff>:/images/screens/set.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>48</width>
      <height>56</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="homeBtn">
    <property name="geometry">
     <rect>
      <x>772</x>
      <y>343</y>
      <width>49</width>
      <height>52</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background: transparent;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="../resources/images/images.qrc">
      <normaloff>:/images/screens/reset.png</normaloff>:/images/screens/reset.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>49</width>
      <height>52</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="aiBtn">
    <property name="geometry">
     <rect>
      <x>762</x>
      <y>409</y>
      <width>62</width>
      <height>54</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background: transparent;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="../resources/images/images.qrc">
      <normaloff>:/images/screens/AI.png</normaloff>:/images/screens/AI.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>62</width>
      <height>54</height>
     </size>
    </property>
   </widget>
   <widget class="QLabel" name="backgroundLbl">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1180</width>
      <height>728</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="../resources/images/images.qrc">:/images/screens/form_main.jpg</pixmap>
    </property>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>42</y>
      <width>676</width>
      <height>577</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="cardGL">
     <property name="sizeConstraint">
      <enum>QLayout::SetDefaultConstraint</enum>
     </property>
     <property name="spacing">
      <number>15</number>
     </property>
    </layout>
   </widget>
   <widget class="QLabel" name="evilLbl">
    <property name="geometry">
     <rect>
      <x>791</x>
      <y>482</y>
      <width>298</width>
      <height>155</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="../resources/images/images.qrc">:/images/screens/unluckfloor.png</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="gainLbl">
    <property name="geometry">
     <rect>
      <x>969</x>
      <y>220</y>
      <width>59</width>
      <height>46</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="../resources/images/images.qrc">:/images/screens/credit.png</pixmap>
    </property>
   </widget>
   <widget class="QPushButton" name="nextFloorBtn">
    <property name="geometry">
     <rect>
      <x>298</x>
      <y>231</y>
      <width>99</width>
      <height>99</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(255, 255, 255, 0);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="../resources/images/images.qrc">
      <normaloff>:/images/screens/button_next.png</normaloff>:/images/screens/button_next.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>99</width>
      <height>99</height>
     </size>
    </property>
   </widget>
   <zorder>backgroundLbl</zorder>
   <zorder>exitBtn</zorder>
   <zorder>newGameBtn</zorder>
   <zorder>pauseBtn</zorder>
   <zorder>resumeBtn</zorder>
   <zorder>amPB</zorder>
   <zorder>hpPB</zorder>
   <zorder>heroBtn</zorder>
   <zorder>levelPB</zorder>
   <zorder>statusLbl</zorder>
   <zorder>levelLbl</zorder>
   <zorder>userLbl</zorder>
   <zorder>bestLbl</zorder>
   <zorder>savedLbl</zorder>
   <zorder>goldLbl</zorder>
   <zorder>shopBtn</zorder>
   <zorder>jumpCostLbl</zorder>
   <zorder>homeCostLbl</zorder>
   <zorder>aiCostLbl</zorder>
   <zorder>infoTB</zorder>
   <zorder>jumpBtn</zorder>
   <zorder>homeBtn</zorder>
   <zorder>aiBtn</zorder>
   <zorder>gridLayoutWidget</zorder>
   <zorder>evilLbl</zorder>
   <zorder>gainLbl</zorder>
   <zorder>nextFloorBtn</zorder>
  </widget>
 </widget>
 <resources>
  <include location="../resources/images/images.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>exitBtn</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>941</x>
     <y>686</y>
    </hint>
    <hint type="destinationlabel">
     <x>589</x>
     <y>363</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
